src/test/java/blackjack/domain/card/BlackjackScoreTest.java

```diff
@@ -0,0 +1,201 @@
+package blackjack.domain.card;
+
+import static blackjack.fixture.CardFixture.CLUB_NINE;
+import static blackjack.fixture.CardFixture.CLUB_TEN;
+import static blackjack.fixture.CardFixture.DIAMOND_ACE;
+import static blackjack.fixture.CardFixture.DIAMOND_KING;
+import static blackjack.fixture.CardFixture.DIAMOND_NINE;
+import static blackjack.fixture.CardFixture.DIAMOND_TEN;
+import static blackjack.fixture.CardFixture.DIAMOND_THREE;
+import static blackjack.fixture.CardFixture.DIAMOND_TWO;
+import static blackjack.fixture.CardFixture.HEART_NINE;
+import static blackjack.fixture.CardFixture.HEART_TEN;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class BlackjackScoreTest {
+
+    @Test
+    void 승부를_보는_두명이_모두_21을_초과한_경우_무승부이다() {
+        //given
+        Cards mainCards = new Cards(
+                CLUB_TEN,
+                DIAMOND_TEN,
+                HEART_TEN
+        );

```

Comment:
```suggestion
        Cards mainCards = new Cards(CLUB_TEN, DIAMOND_TEN, HEART_TEN);

```
굳이 내려서 메서드 길이를 늘릴 필요는 없다고 생각합니다.